lte/gateway/python/integ_tests/s1aptests/s1ap_utils.py

```diff
@@ -1025,90 +1026,109 @@ def _wait_for_pipelined_to_initialize(self):
         WAIT_INTERVAL_SECONDS = 5
         MAX_WAIT_SECONDS = 120
         print(f"  check every {WAIT_INTERVAL_SECONDS} seconds (max {MAX_WAIT_SECONDS} seconds) if pipelined is started ...")
-        pipelined_is_running = False
         datapath_is_initialized = False
         while not datapath_is_initialized:
-            time.sleep(WAIT_INTERVAL_SECONDS)
-            wait_time_seconds += WAIT_INTERVAL_SECONDS
```

Comment:
This change was adopted as part of PR https://github.com/magma/magma/pull/13980. Is this change needed here?

Reply:
We switched the loop behavior to check the `pipelined`/`datapath` status before the sleep to reduce the overall wait time in the case of a containerized AGW. The check is called after all containers report healthy, with `mme` usually being the last one to do so. Chances are, that we already have a running `pipelined`/`datapath` and waiting first would be unnecessary.